MoviesApp/Core/AssemblyMovie.swift

```diff
@@ -26,7 +32,19 @@ final class AssemblyMovie: AssemblyMovieProtocol {
     func createMovieDetailModule(movieId: Int?, router: RouterMoviProtocol) -> UIViewController {
         let view = MovieDetailViewController()
         let moviAPIService = MovieAPIService()
-        let presenter = MovieDetailViewPresenter(view: view, moviAPIService: moviAPIService, movieID: movieId)
+        let entityMapper = MovieInfoEntityMapper()
+        let contextProvider = DBContextProvider()
+        let imageAPIService = ImageAPIService()
+        let cacheImageService = CacheImageService()
+        let loadImageProxy = LoadImageProxy(service: imageAPIService, cacheImageService: cacheImageService)
```

Comment:
```suggestion
let imageAPIService = ImageAPIService()
         let cacheImageService = CacheImageService()
        let loadImageProxy = LoadImageProxy(service: imageAPIService, cacheImageService: cacheImageService)
```
тоже выносим и инкапсулируем в ImageService как в тз!